utils/pageTeamsFilterFunctions.js

```diff
@@ -0,0 +1,45 @@
+const { env } = require('../config.js');
+const config = require('../config.js')[env];
+const bdk = require('@salesforce/refocus-bdk')(config);
+
+/**
+ * @param {string} name - name of function where error occurred.
+ * @param {any} returnValue - value to be returned after logging.
+ * @returns {any} - whatever was passed as the returnsValue param.
+ */
+function logInvalidArgs(name, returnValue) {
+  bdk.log.error(`Invalid args passed to function ${name}`);
+  return returnValue;
+}
+
+/**
+ * @param {Array<Incident>} incidentList - list of pager incidents that have already occurred.
+ * @param {Array<Team>} teamsToPage - list of teams to autopage, from room settings.
+ * @returns {Array<Team>} - list of teams from teamsToPage which have not been already paged.
+ */
+function removeAlreadyPagedTeams(incidentList, teamsToPage) {
+  if (!incidentList || !teamsToPage?.length)
```

Comment:
maybe either keep the return on the one liner or add the brackets? just for readability(just a picky comment). 
coming from python I don't mind the indentation by spaces 😛